### ISSUE TITLE
Bulk operation for scheduling publishing of multiple pieces of content.

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#3115]
Follow-up for #857... as per issue title.

**Comment by quicksketch** [comment]
I took an initial stab at this in https://github.com/backdrop/backdrop/pull/2192.

One awkward situation I encountered is what to do about _already published_ content. We could just ignore it and not schedule it at all, since if it's already published there is nothing to be done. For now I've added a checkbox for "Unpublish any published posts" (weird wording I know). So if you want to bulk schedule content you don't have to first unpublish things. The checkbox only shows up if you have selected any already published content.

Screenshot:

![image](https://user-images.githubusercontent.com/100206/40039204-42c3e314-57ca-11e8-8da4-e6fa3f9eda69.png)

**Comment by olafgrabienski** [comment]
There's a typo in the screenshot:

> If content is already published, it will not **Sbe** scheduled.

**Comment by laryn** [comment]
@quicksketch Yes, I ran into that awkward situation in Save Draft as well. In that case, I am not showing the schedule option for already published content. So there are two tracks: on new/unpublished content you can save draft/publish/schedule, but on already published content you can update/unpublish. 

That is tougher to do on a bulk form with so many extra variables/pages and I like your solution.
